src/components/Dropdown/components/Label/index.tsx

```diff
@@ -16,22 +16,34 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { ReactElement, useMemo } from 'react'
+import { ReactElement, ReactNode, isValidElement, useMemo } from 'react'
 import classNames from 'classnames'
 
 import { DropdownItem, ItemLayout } from '../../props'
+import { Tag } from '../../../Tag'
 import styles from '../../dropdown.module.css'
 
 const LABEL_DIVIDER = '·'
 
 export type LabelProps = {
   layout: ItemLayout
   item: DropdownItem
+  tag?: ReactNode
+}
+
+const validateTagType = (tag?: ReactNode): void => {
+  if (!tag) {
+    return
+  }
+  if (!isValidElement(tag) || tag.type !== Tag) {
+    throw new Error('`tag` must be a Tag component')
```

Comment:
```suggestion
    throw new Error('`item.tag` must be a Tag component')
```

also it would be helpful for developers to provide the item id in the log, so it's easier to find out the misconfigured item

Reply:
agree, done